internal/rbd/nodeserver.go

```diff
@@ -252,7 +256,7 @@ func populateRbdVol(
 		// fallback to rbd-nbd,
 		rv.Mounter = rbdNbdMounter
 	} else {
-		rv.Mounter = req.GetVolumeContext()["mounter"]
+		rv.Mounter = mounter
```

Comment:
Shouldn't `rv.Mounter` be already set here regardless because of
```
	if rbdVol.Mounter, ok = volOptions["mounter"]; !ok {
		rbdVol.Mounter = rbdDefaultMounter
	}
```
in `genVolFromVolumeOptions`?

Reply:
Yes, correct, this is doing the same thing. removed it now.